skimage/filters/_gaussian.py

```diff
@@ -19,8 +19,8 @@ def _convert_input(image, preserve_range):
 
 
 def gaussian(image, sigma=1, output=None, mode='nearest', cval=0,
-             multichannel=None, preserve_range=False):
-    """Multi-dimensional Gaussian filter
+             multichannel=None, preserve_range=False, truncate=4.0):
```

Comment:
I'm adding this to my growing list of things we wouldn't have to worry about if we used Python 3.5. (With keyword-only arguments.) Thanks! ;)

fwiw I agree that it's better to just put the kwarg at the very end and forget about the horrible deprecation cycle.